### DISCUSSION TITLE
Request for Addition: Wallet Icon

### DISCUSSION THREAD

**Discussion by sntgchns** [microsoft/microsoft-ui-xaml#8856] [Ideas]
Hey team,

I noticed that Segoe Fluent Icon set currently don't have a wallet icon, and I believe it would be a valuable addition. Wallet icons are quite common in various financial and e-commerce applications, and having one readily available in our collection would be beneficial for our users.

Could we please consider adding a wallet icon to our set? I'd be happy to provide some design suggestions or even contribute to the design if needed.

Example: Google's Material Icon:
 `<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" crossorigin="anonymous"><span class="material-icons-outlined">account_balance_wallet</span>`

Looking forward to hearing your thoughts on this!

Best regards,

**Comment by Poopooracoocoo** [comment]
It is part of [FluentUI System Icons](https://github.com/microsoft/fluentui-system-icons/issues/346#issuecomment-956935812) at least

**Comment by yinhx3** [comment]
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/45716406/0a6131ff-8307-49cd-af8d-3cb3afa743de)
[FluentUI System Icons](https://github.com/microsoft/fluentui-system-icons?tab=readme-ov-file#fluent-ui-system-icons) have this icon, but the font file in Windows11 isn't complete.
You can use [this NuGet package](https://www.nuget.org/packages/KitUIN.FluentIcon) to get the icon.
